src/components/Switch/switch.component.css

```
@@ -0,0 +1,60 @@
+html {
+  font-size: 62.5%;
+}
+
+:root {
+  --main-color: #6200ee;
+  --secondary-color: #c39ef8;
+}
+
+.inputWrapper {
+  position: relative;
+  top: 0.5rem;
+  left: 0.5rem;
+}
+
+.input {
+  position: absolute;
+  left: -9999rem;
+  top: -99999rem;
+}
+
+.slider {
+  display: flex;
+  cursor: pointer;
+  width: 3.5rem;
+  height: 1.5rem;
+  border-radius: 50%;

```

TIL (today I learned), 50% is not the same as a large `px` value. Sorry about that, my expectation was that this would work the same, but it does not.
```suggestion
  border-radius: 7px;
```

---

As far as I know, the % takes its value from the parent element. So yeah I made it in px.

---

I notice this change hasn't been applied yet